java/openjdk-java/portfile

```diff
@@ -0,0 +1,380 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+PortSystem          1.0
+name                openjdk-java
+version             17
+categories          java
+platforms           darwin
+description         an Open Source, Open Innovation, Open Standards implementation of the Java Platform, Standard Edition, and related projects.
+homepage            openjdk.java.net
+license             GPL-2+CE
+universal_variant   no
+
+set long_description_openjdk-ga \
+ Production-ready open-source builds of the Java Development Kit
+
+set long_description_openjdk-ea \
+ early-access, open-source builds
+
+set long_description_openjfx \
+ JavaFX is an open source, next generation client application platform for desktop, mobile and embedded systems built on Java. It is a collaborative effort by many individuals and companies with the goal of producing a modern, efficient, and fully featured toolkit for developing rich client applications.
+
+set obsoleted_ports {
+openjdk-java
+}
+
+if {${subport} in ${obsoleted_ports}} {
+    PortGroup    obsolete 1.0
+
+}
+
+if {${subport} eq "openjdk-java"} {
+    # The openjdk-java is not installable, but recommends users to install the latest java subport
+    replaced_by  openjdk${version}
+}
+
+subport openjdk-6-bsd-i386-soylatte {
+ version             1.6
+ set build           1.0.3
+ description         openjdk
+ supported archs     i386
+ long_description    openjdk 6
+ master_sites        https://landonf.org/static/soylatte/bsd-dist/javasrc_${version}_jrl_darwin/soylatte16-i386-${build}.tar.bz2
```

Comment:
This can't work. `master_sites` must point to the directory the file is in, excluding the filename itself.

Reply:
Ok

Reply:
Pointing master sites to the directory the file is in